docs/guides/stonesense-art-guide.rst

```diff
@@ -0,0 +1,63 @@
+.. _stonesense-art-guide:
+
+Stonesense art creation guide
+=============================
+
+Understanding sprites
+---------------------
+Understanding how Stonesense deals with sprites is central to anyone who wishes to modify the content.
+The scheme is not very complicated, and this guide will give a short introduction to how they work.
+The way sprites are loaded is fairly generalized. With the exception of floors, which we will discuss later,
+all sprites are 32x32 pixels big and come in groups known as Sprite Sheets. All sprites are loaded and
+rendered in 32-bit full-color PNGs.
+
+
+.. image:: ../images/stonesense-sprite-sample.png
+    :align: left
+
+Here's an example of a typical Stonesense sprite.
+
+Note that, in order not to conflict with neighboring sprites, a sprite must actually be within a smaller
+area than its 32x32 block.
+A template for the area used by most sprites is:
+
+.. image:: ../images/stonesense-sprite-template.png
+    :align: left
+
+The solid area is the floor space taken up by a sprite, while the dotted box indicates the volume above this
+area corresponding to one z-level.
+
+Sprite sheets
+-------------
+There can be an arbitrary number of sprite sheets configured for Stonesense, yet some are always present as
```

Comment:
"yet some are always present" implies that there are conditions where some sprite sheets are not used. What are those conditions?